conduit/plugins/importers/filereader/test_resources/conduit_data/conduit.yml

```diff
@@ -0,0 +1,72 @@
+# Log verbosity: PANIC, FATAL, ERROR, WARN, INFO, DEBUG, TRACE
+log-level: INFO
```

Comment:
I'd consider making this programmatically, or removing as much as possible so that there's less opportunity for things to become stale.

Reply:
d7fd94890b1b6fa5dc7bd8b54eb809f5f1abcca6